apps/backend/src/services/compose_deployment.py

```
@@ -213,34 +211,34 @@ async def scan_available_ports(self, request: PortScanRequest) -> PortScanResult
             await self._scan_docker_ports(request.device, result)
-            
+
             # Remove Docker ports from system ports to avoid duplication
             docker_ports = set(result.docker_port_usage.keys())
             for port in docker_ports:
                 result.system_port_usage.pop(port, None)
-            
+
             # Focus on used ports only - combine system and Docker ports
             all_used_ports = set(result.system_port_usage.keys()) | docker_ports
             result.used_ports = sorted(list(all_used_ports))
```

Ruff (0.12.2)

Unnecessary `list()` call within `sorted()`

Remove the inner `list()` call

(C414)